.circleci/config.yml

```diff
@@ -44,6 +44,31 @@ jobs:
                 field_value: << parameters.goversion >>
 
 workflows:
+  nightly:
+    triggers:
+      - schedule:
+          cron: "0 0 * * *"
+          filters:
+            branches:
+              only:
+                - main
+    jobs:
+      - setup
+      - watch:
+          requires:
+            - setup
+      - test_libhoney:
```

Comment:
I think it would help to have the go version as part of the job title to see which version failed.

Reply:
Looking at the Python PR, would it be possible to use a matrix to handle the go version instead?

Reply:
Forget this for now, we can tidy up configs later 👍